new-lamassu-admin/src/components/LogsDownloaderPopper.js

```diff
@@ -121,53 +121,71 @@ const styles = {
 
 const useStyles = makeStyles(styles)
 
-const LogsDownloaderPopover = ({ id, open, anchorEl, onClose, logsResponse, ...props }) => {
-  const [radioButtons, setRadioButtons] = useState(0)
+const LogsDownloaderPopover = ({ id, name, open, anchorEl, getTimestamp, logs, title, ...props }) => {
+  const radioButtonOptions = [{ label: 'All logs', value: 'all' }, { label: 'Date range', value: 'range' }]
```

Comment:
Maybe create consts for `all` and `range`, that way you can reference that in code instead of `radioButtonOptions[0].value`.